src/helpers/validation.js

```diff
@@ -0,0 +1,13 @@
+import { REGEX } from '../constants/constants';
+
+export const validateName = (name) => {
```

Comment:
validateName and validateImageUrl are pretty much doing the same thing. Make a generic one (validateInputField) and send the REGEX constant  and textInput as parameters.